lusidtools/cocoon/config/domain_settings.json

```diff
@@ -59,5 +59,16 @@
       "full_key_format": false,
       "required_call_attributes": [],
       "unique_attributes": []
+  },
+
+    "portfolio_group": {
+      "domain": "portfoliogroups",
+      "batch_allowed": true,
+      "default_batch_size": 2000,
+      "top_level_model": "CreatePortfolioGroupRequest",
+      "portfolio_specific": true,
+      "full_key_format": false,
+      "required_call_attributes": ["scope"],
```

Comment:
code and effective_at are missing from the required_call_attributes list in the config here.

Reply:
Sure. updated.

Reply:
`effective_at` is required for add_portfolio_to_group and not create_portfolio_group, so I have not added `effective_at `to the domain config.